python/lsst/ts/dimm/dimm_csc.py

```diff
@@ -368,6 +368,59 @@ async def seeing_loop(self):
                 )
                 break
 
+    async def do_gotoAltAz(self)
```

Comment:
In all the new commands the `data` parameter is missing from the function definition. And the definitions need to be terminated with a colon. Please change to `(self, data):`

Reply:
ok